assets/js/swedbank-pay-design-guide-theme.js

```diff
@@ -11,37 +12,38 @@
         flowchart: {
             useMaxWidth: false
         }
-    });
+    };
+    mermaid.initialize(configObject);
 
-    mermaid.init(undefined, 'code.language-mermaid');
+    mermaid.init(undefined, "code.language-mermaid");

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5600350)